client/components/gsuite/gsuite-price/index.jsx

```diff
@@ -1,30 +1,100 @@
 /**
  * External dependencies
  */
+import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { useTranslate } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
+import { abtest } from 'lib/abtest';
 import { getAnnualPrice, getMonthlyPrice } from 'lib/gsuite';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
+/**
+ * Determines whether a discount can be applied to the specified product via a sales coupon.
+ *
+ * @param {Object} product - G Suite product
+ * @returns {Boolean} - true if a discount can be applied, false otherwise
+ */
+const hasValidDiscount = ( product ) => {
+	if (
+		! product ||
+		! product.sale_cost ||
+		! product.sale_coupon ||
+		! product.sale_coupon.start_date ||
+		! product.sale_coupon.expires
+	) {
+		return false;
+	}
+
+	const currentTime = Date.now();
+	const startDate = new Date( product.sale_coupon.start_date );
+	const endDate = new Date( product.sale_coupon.expires );
+
+	return currentTime >= startDate && currentTime <= endDate;
+};
+
 const GSuitePrice = ( { currencyCode, product } ) => {
 	const translate = useTranslate();
 
 	const cost = product?.cost ?? null;
-	const annualPrice = cost && currencyCode ? getAnnualPrice( cost, currencyCode ) : '-';
-	const monthlyPrice = cost && currencyCode ? getMonthlyPrice( cost, currencyCode ) : '-';
+
+	const annualPrice = getAnnualPrice( cost, currencyCode );
+	const monthlyPrice = getMonthlyPrice( cost, currencyCode );
+
+	if ( abtest( 'gsuitePrice' ) === 'discount' ) {
+		const isDiscounted = hasValidDiscount( product );
+
+		return (
+			<div className="gsuite-price">
+				<h4 className="gsuite-price__monthly-price">
+					{ translate( '{{strong}}%(price)s{{/strong}} per user/month', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( '{{strong}}%(price)s{{/strong}} per user / month' )` *ES Score: 8*